assembl/static2/js/app/components/debate/survey/navigation.jsx

```diff
@@ -0,0 +1,151 @@
+    const offsetArray = [];
+    for (let i = 0; i < questionsLength; i += 1) {
+      const questionOffset = Number(getDomElementOffset(document.getElementsByClassName('question-title')[i]).top);
+      offsetArray.push(questionOffset);
+    }
+    return offsetArray;
+  }
+  displayNav() {
+    const proposals = document.getElementById('proposals');
+    const proposalsOffset = getDomElementOffset(proposals).top;
+    const firstTextarea = document.getElementsByClassName('txt-area')[0];
+    if (!firstTextarea) {
+      return;
+    }
+    const limitToHide = getDomElementOffset(firstTextarea).top + firstTextarea.clientHeight;
+    const limitToShow = limitToHide + document.getElementById('nav').clientHeight;
+    const windowOffset = window.pageYOffset + window.innerHeight;
+    const { debateData } = this.props.debate;
+    const isPhaseCompleted = getIfPhaseCompletedByIdentifier(debateData.timeline, 'survey');
+    if (windowOffset < limitToHide && !isPhaseCompleted) {
+      this.setState({
+        isHidden: true
+      });
+    }
+    if (windowOffset > limitToShow && !isPhaseCompleted) {
+      this.setState({
+        isHidden: false
+      });
+    }
+    if (windowOffset > proposalsOffset) {
+      this.setState({
+        navPosition: 'absolute',
+        navBottom: proposals.clientHeight
+      });
+    } else {
+      this.setState({
+        navPosition: 'fixed',
+        navBottom: 0
+      });
+    }
+  }
+  displayPagination() {
+    const questionsOffset = this.getQuestionsOffset(this.state.questionsLength);
+    const windowOffset = window.pageYOffset + window.innerHeight;
+    let currentQuestionNumber = 0;
+    for (let i = 0; i < this.state.questionsLength; i += 1) {
+      if (windowOffset > questionsOffset[i]) {
+        currentQuestionNumber = i + 1;
+      }
+    }
+    this.setState({
+      currentQuestionNumber: currentQuestionNumber
+    });
+  }
+  scrollToNextQuestion() {
+    const navbarHeight = document.getElementById('timeline').clientHeight;
+    const target = document.getElementById(`q${this.state.currentQuestionNumber + 1}`);
+    const targetOffset = Number(getDomElementOffset(target).top) + navbarHeight;
+    scrollToElement(document.body, targetOffset, 600);
+  }
+  scrollToPreviousQuestion() {
+    const navbarHeight = document.getElementById('timeline').clientHeight;
+    const target = document.getElementById(`q${this.state.currentQuestionNumber - 1}`);
+    const targetOffset = Number(getDomElementOffset(target).top) + navbarHeight;
+    scrollToElement(document.body, targetOffset, 600);
+  }
+  render() {
+    const barWidth = calculatePercentage(this.state.currentQuestionNumber, this.state.questionsLength);
+    return (
+      <section
+        className={this.state.isHidden ? 'hidden' : 'shown navigation-section background-color'}
+        id="nav"
+        style={{ position: this.state.navPosition, bottom: this.state.navBottom }}
+      >
```

Comment:
@paolina92 I meant the `>`, it sits on it's own in line with the start of the `<section` tag.